compiler/src/typed/env.re

```diff
@@ -2278,30 +2210,33 @@ let build_signature_with_imports =
     cmi_type_metadata: type_metadata,
     cmi_config_sum: Cmi_format.config_sum(),
   };
+};
 
+let add_cmi_to_persistent_structures = (modname, filename, cmi) => {
```

Comment:
It looks like the `modname` here isn't used anywhere, so we should be able to remove it from here, and the anftree, right?

Reply:
Yep good point, this was from a previous iteration of this PR that I ended up restructuring